src/openrct2/ride/Ride.cpp

```diff
@@ -891,7 +927,9 @@ void ride_init_all()
 void reset_all_ride_build_dates()
 {
     for (auto& ride : GetRideManager())
-        ride.build_date -= gDateMonthsElapsed;
+    {
+        ride.build_date = 0;
```

Comment:
Doesn't this reset the build date to 1 March, Year 1?

Reply:
Okay, now I understand what the code was trying to do.

Reply:
Turns out this method is only called in scenario_start where `gDateMonthsElapsed` is set to 0 anyway. I have fixed it nonetheless.